contrib/babelfishpg_tds/src/backend/tds/tdsutils.c

```diff
@@ -48,7 +48,7 @@ void tdsutils_ProcessUtility (PlannedStmt *pstmt, const char *queryString, bool
 ProcessUtility_hook_type next_ProcessUtility = NULL;
 static void call_next_ProcessUtility (PlannedStmt *pstmt, const char *queryString, bool readOnlyTree, ProcessUtilityContext context, ParamListInfo params, QueryEnvironment *queryEnv, DestReceiver *dest, QueryCompletion *completionTag);
 static void check_babelfish_droprole_restrictions(char *role);
-static void check_babelfish_renamerole_restrictions(char *role);
+static void check_babelfish_alterrole_restictions(char *role, bool allow_alter_operation);
```

Comment:
I think `allow_alter_role_operation` would be better.

Reply:
Will fix in next revision